lib/Context.php

```diff
@@ -120,19 +122,28 @@ public function setCacheDir(string $cacheDir = null)
      *
      * @throws InvalidResponseException
      */
-    public function makeRequest(string $path, string $method, $params = null): ResponseInterface
-    {
+    public function makeRequest(
+        string $path,
+        string $method,
+        $params = null
+    ): ResponseInterface {
         $this->refreshToken();
 
-        $request = $this->messageFactory->createRequest($method, self::BASE_URI.$path, [
-            'Content-Type' => 'application/json',
-            'Authorization' => 'Bearer '.$this->token->getAccessToken(),
-        ], json_encode($params));
+        $request = $this->messageFactory->createRequest(
+            $method,
+            self::BASE_URI.$path,
+            [
+                'Content-Type'  => 'application/json',
+                'Authorization' => 'Bearer '.$this->token->getAccessToken(),
+            ],
+            json_encode($params)
+        );
 
         $response = $this->client->sendRequest($request);
         if (200 !== $response->getStatusCode()) {
-            $responseBody = (string)$response->getBody();
-            $message = "Response not OK when requesting an access token. Response body: $responseBody";
+            $responseBody = (string) $response->getBody();
+            $message = "Response not OK when requesting an access token. "
+                     . "Response body: {$responseBody}";
```

Comment:
do not break a line (event strings) if it stays within the soft limit of PSR-2 (120 chars, http://www.php-fig.org/psr/psr-2/#overview). 

Also, keep the operator in the line of the first argument. In this case it should be (but it is not for what mentioned above):
```php
$message = "Response not OK when requesting an access token. "  .
     "Response body: {$responseBody}"
;
```